### PR TITLE
[General] Add links to YouTube videos

### PR DESCRIPTION
**Pull request by ehumph**
This PR creates embedded videos for each section (introductory and summary videos). The embedded links point to the videos posted on YouTube.

### CODE REVIEW

**Comment by avahoffman** [comment]
Can you remove the `:` and `-` characters from the h1 titles? It makes Leanpub render kind of funky.

File: 07-bio_code.Rmd [hunk 1]
```diff
@@ -54,8 +54,16 @@ When using AI to help generate code examples, perform refactoring, understand co
 :::
 
 
-## Summary
+# SUMMARY VIDEO: AI for Bioinformatics - The Main Points!
```
Comment (avahoffman) [inline 07-bio_code.Rmd:57]:
```suggestion
# SUMMARY VIDEO AI for Bioinformatics the Main Points
```

Comment (ehumph) [reply 07-bio_code.Rmd:57]: Damn, I forgot about that one

**Review by avahoffman — APPROVED** [review]
LGTM!

**Pull request merged**